.gitpod.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gitpod/workspace-full-vnc:latest
```

Comment:
I don't want to include this file. If there is a simple GitPod setup, I am happy to maintain it in Carbon.

It is my understanding from what @anudeepreddy  said here that all we need to do is update the node version our CI runs on.

Reply:
hey @mfix22 , I think there has been a misunderstanding. I just wanted to know if we could run the development setup on node 10.x as your CI was using the same. 

As you have clearly mentioned that you want all the development to happen in node 12.x, this is the PR that @nisarhassan12 could come up with. If you totally don't want to add the .gitpod.Dockerfile to the setup, It's totally fine. I will try to get in touch with the team at Gitpod and try to find an alternate solution with just the .gitpod.yml file. Thank you for your patience, It's highly appreciated.

Reply:
@anudeepreddy understood. You are correct, I do not want to support a custom Dockerfile for Gitpod — it should work appropriately on Node's current LTS version at the very least. 

I will reopen #963